src/other-scripts/recaptcha/index.js

```diff
@@ -139,11 +140,24 @@ function renderWidget( form ) {
 			refreshWidget( button );
 		};
 
+		const errorCallback = () => {
+			refreshWidget( button );
+			const message = wp.i18n.__( 'There was an error with reCAPTCHA. Please try again.', 'newspack-plugin' );
+			if ( errorHandler ) {
+				errorHandler( message );
+			} else {
+				// eslint-disable-next-line no-alert
+				alert( message );
```

Comment:
I decided to ignore the eslint rule and trigger an alert here because the scripts default behavior is to alert with a generic connection error message.